tests/jenkins/TestDockerCopy.groovy

```diff
@@ -60,8 +60,27 @@ class TestDockerCopy extends BuildPipelineTest {
         assertJobStatusSuccess()
 
         // Ensure the gcrane is executed in an external shell script exactely once
-        def copyContainerCommand = getCommands('docker').findAll {
-            shCommand -> shCommand.contains('gcrane cp opensearchstaging/opensearch:1.3.2 opensearchproject/opensearch:1.3.2; docker logout')
+        def copyContainerCommand = getCommands('crane').findAll {
+            shCommand -> shCommand.contains('set -x && crane cp opensearchstaging/ci-runner:testtag public.ecr.aws/opensearchstaging/ci-runner:testtag2')
```

Comment:
`gcrane` to renamed to `crane`?

Reply:
See explanations: https://github.com/opensearch-project/opensearch-build-libraries/pull/187#issuecomment-1541028643